### ISSUE TITLE
#slide-out scrollbar isn't perfect in Firefox 53

### ISSUE THREAD

**Issue by dmytrokyrychuk** [viewflow/docs#3]
3rd party integration menu item in the sidebar may not be visible when the window height is too small. This is particularly noticeable on pages with a lot of menu items (such as Core concepts).

On the picture below the scrollbar is in its lowest position, but the last menu item is not visible.
![documentation - mozilla firefox_343](https://user-images.githubusercontent.com/699961/27011507-9d38c774-4ec5-11e7-9371-565c8b772931.png)

I've been able to fix this issue using the following Stylish style:

```
@namespace url(http://www.w3.org/1999/xhtml);

@-moz-document domain("viewflow.io") {
  #slide-out {
    height: 100vh;
  }
}
```

The scrollbar appears correctly in Google Chrome 59.

**Comment by kmmbvnr** [comment]
Yeah, thank for the report! Django-material adds nasty hack - the last empty menu item to fix that.  

Few days ago I saw the commit that fixes some sidebar problems in the MaterializeCSS repository. I'll check this issue when next version of MaterializeCSS would be available.

**Comment by dmytrokyrychuk** [comment]
The issue is reproducible newer Firefox 58 as well.
